R/delineate.R

```diff
@@ -0,0 +1,53 @@
+#' Delineate a corridor around a river.
+#'
+#' @param city_name A place name as a string
+#' @param river_name A river name as a string
+#' @param crs The projected Coordinate Reference System (CRS) to use. If not
+#'   provided, the suitable Universal Transverse Mercator (UTM) CRS is selected
+#' @param bbox_buffer Add a buffer region to the city boundary to avoid edge
+#'   effects close to its limits
+#' @param initial_method The method employed to define the initial river
+#'   corridor geometry. See [initial_corridor()] for the available methods
```

Comment:
```suggestion
#'   corridor geometry. See \code{\link{initial_corridor}} for the available methods
```

Reply:
I noticed these simpler links could be used in roxygen: https://cran.r-project.org/web/packages/roxygen2/vignettes/rd-formatting.html#default-link-text 

And indeed the link seems to be properly resolved on the man page: https://github.com/CityRiverSpaces/CRiSp/blob/28cb78e44f79d09b25265cb7c1fbe0d07b6842be/man/delineate_corridor.Rd#L30

Reply:
Ah, nice! Then please ignore these link formatting suggestions.